lib/vm/CMakeLists.txt

```diff
@@ -14,13 +14,6 @@ set(ssvmLibs
   ssvmHostModuleSSVMProcess
 )
 
-if (NOT SSVM_DISABLE_AOT_RUNTIME)
```

Comment:
Why do you remove this?
We still need a way to disable aot build.

Reply:
Since `vm` package is not related to aot component, we can remove this unused condition.